Packages/.swiftlint.yml

```diff
@@ -20,6 +20,8 @@ disabled_rules:
   - file_length
   - syntactic_sugar
   - unused_capture_list
+  - nesting
+  - large_tuple
```

Comment:
Can we add these to our other .swiftlint.yml configs as well for consistency? 
(we can add some automation around this in the future)

Reply:
I will open PRs for these.

As an aside, I wonder if there is a way to separate the rules part of SwiftLint config (which we could share across projects) from the project-specific part of config (ignored files, etc.)